### PR TITLE
KSTAT-453 Update to ES6

### PR DESCRIPTION
**Pull request by RobLoach**
#453

### CODE REVIEW

File: src/kalastatic.js [hunk 1]
```diff
@@ -17,9 +18,9 @@ function KalaStatic(nconf) {
     source: 'src',
     destination: 'build',
     plugins: [
-      // Load information from the environment variables.
+      // Load information from the environment letiables.
       'metalsmith-env',
-      // Define any global variables.
+      // Define any global letiables.
```
Comment (RobLoach) [inline src/kalastatic.js]: letiables aren't a thing.

File: src/kalastatic.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-var path = require('path')
-var kss = require('kss/lib/cli')
-var Metalsmith = require('metalsmith')
-var extend = require('extend')
+
+const assert = require('assert')
+const path = require('path')
+const kss = require('kss/lib/cli')
+const Metalsmith = require('metalsmith')
+const extend = require('extend')
```
Comment (thiagodemellobueno) [inline src/kalastatic.js]: Should we switch to es6 imports?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

**Comment by thiagodemellobueno** [comment]
raaaad

File: src/kalastatic.js (hunk 2 above)
Comment (RobLoach) [inline src/kalastatic.js]: Yesssssssssss

Comment (RobLoach) [inline src/kalastatic.js]: Looks like `import` will only work in Node.js harmony mode. Let's keep it out for now.

**Pull request merged**